tests/test_integration_async.py

```diff
@@ -0,0 +1,617 @@
+
+        with self.settings(INTERNAL_IPS=FailOnIteration()):
+            response = await self.async_client.get("/regular/basic/")
+            self.assertEqual(response.status_code, 200)
+            self.assertContains(response, "djDebug")  # toolbar
+
+    # async def test_should_render_panels_RENDER_PANELS(self):
+    #     """
+    #     The toolbar should force rendering panels on each request
+    #     based on the RENDER_PANELS setting.
+    #     """
+    #     toolbar = DebugToolbar(self.request, self.get_response)
+    #     self.assertFalse(toolbar.should_render_panels())
+    #     toolbar.config["RENDER_PANELS"] = True
+    #     self.assertTrue(toolbar.should_render_panels())
+    #     toolbar.config["RENDER_PANELS"] = None
+    #     self.assertTrue(toolbar.should_render_panels())
+
+    # def test_should_render_panels_multiprocess(self):
+    #     """
+    #     The toolbar should render the panels on each request when wsgi.multiprocess
+    #     is True or missing.
+    #     """
+    #     request = rf.get("/")
+    #     request.META["wsgi.multiprocess"] = True
+    #     toolbar = DebugToolbar(request, self.get_response)
+    #     toolbar.config["RENDER_PANELS"] = None
+    #     self.assertTrue(toolbar.should_render_panels())
+
+    #     request.META["wsgi.multiprocess"] = False
+    #     self.assertFalse(toolbar.should_render_panels())
+
+    #     request.META.pop("wsgi.multiprocess")
+    #     self.assertTrue(toolbar.should_render_panels())
+
+    # def _resolve_stats(self, path):
+    #     # takes stats from Request panel
+    #     request = rf.get(path)
+    #     panel = self.toolbar.get_panel_by_id("RequestPanel")
+    #     response = panel.process_request(request)
+    #     panel.generate_stats(request, response)
+    #     return panel.get_stats()
+
+    # def test_url_resolving_positional(self):
+    #     stats = self._resolve_stats("/resolving1/a/b/")
+    #     self.assertEqual(stats["view_urlname"], "positional-resolving")
+    #     self.assertEqual(stats["view_func"], "tests.views.resolving_view")
+    #     self.assertEqual(stats["view_args"], ("a", "b"))
+    #     self.assertEqual(stats["view_kwargs"], {})
+
+    # def test_url_resolving_named(self):
+    #     stats = self._resolve_stats("/resolving2/a/b/")
+    #     self.assertEqual(stats["view_args"], ())
+    #     self.assertEqual(stats["view_kwargs"], {"arg1": "a", "arg2": "b"})
+
+    # def test_url_resolving_mixed(self):
+    #     stats = self._resolve_stats("/resolving3/a/")
+    #     self.assertEqual(stats["view_args"], ("a",))
+    #     self.assertEqual(stats["view_kwargs"], {"arg2": "default"})
+
+    # def test_url_resolving_bad(self):
+    #     stats = self._resolve_stats("/non-existing-url/")
+    #     self.assertEqual(stats["view_urlname"], "None")
+    #     self.assertEqual(stats["view_args"], "None")
+    #     self.assertEqual(stats["view_kwargs"], "None")
+    #     self.assertEqual(stats["view_func"], "<no view>")
```

Comment:
I assume you want to remove all of that? Then do it :)

Reply:
yup I just kept it in order for easy review purpose to let reviewers see what's been involved and what's not.